### DISCUSSION TITLE
Goverment type in /nation

### DISCUSSION THREAD

**Discussion by Random-User-34** [thatguycy/WorldDynamics-Engine#15] [Ideas]
It would be cool if it was set up so the government type appeared in the /nation thing somehow

**Comment by stoleyourharvs** [comment]
This is basically impossible as this would require editing Towny as a plugin, as an alternative, feel free to use /wde nation.

**Reply by 1ByteBit** [reply]
Not sure if this is exactly what the suggestion was, but I think it is possible to add subcommands to towny as shown in https://github.com/TownyAdvanced/Towny/wiki/TownyAPI#adding-your-own-subcommands
If you like, I could test this out myself and make a PR

**Comment by stoleyourharvs** [comment]
Yes, I spoke to LlmDl, if you could do that it would be great! Thanks!
